prody/proteins/interactions.py

```diff
@@ -1921,10 +1921,12 @@ def buildInteractionMatrix(self, **kwargs):
         return InteractionsMap
 
 
-    def showInteractions(self, **kwargs):
+    def showInteractions(self, filename=None, **kwargs):
```

Comment:
I don't think this is the prody way. People can save figures themselves and do more complicated things like subplots too.

Reply:
I did that because in jupyter notebook I was not able to save it. It is probably because of plt.show() which is included in the method.

Reply:
There shouldn't be a plt.show() either. ProDy has a way for checking what figures are open and so on, separately through auto_show. See showEllipsoid or showFractVars for examples

Reply:
Done.